cdk/lib/amiable/__snapshots__/amiable.test.ts.snap

```diff
@@ -1,189 +1,219 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`The Amiable stack matches the snapshot 1`] = `
-Object {
-  "Outputs": Object {
-    "LoadBalancerAmiableDnsName": Object {
+{
+  "Metadata": {
+    "gu:cdk:constructs": [
+      "GuDistributionBucketParameter",
+      "GuGuardianEmailSenderParameter",
+      "GuSESSenderPolicy",
+      "GuAllowPolicy",
+      "GuVpcParameter",
+      "GuSubnetListParameter",
+      "GuSubnetListParameter",
+      "GuPlayApp",
+      "GuCertificate",
+      "GuInstanceRole",
+      "GuSSMRunCommandPolicy",
+      "GuDescribeEC2Policy",
+      "GuLoggingStreamNameParameter",
+      "GuLogShippingPolicy",
+      "GuGetDistributablePolicy",
+      "GuParameterStoreReadPolicy",
+      "GuAmiParameter",
+      "GuHttpsEgressSecurityGroup",
+      "GuAutoScalingGroup",
+      "GuWazuhAccess",
+      "GuApplicationLoadBalancer",
+      "GuStringParameter",
+      "GuApplicationTargetGroup",
+      "GuHttpsApplicationListener",
+      "GuSecurityGroup",
+    ],
+    "gu:cdk:version": "TEST",
+  },
```

Comment:
Added in [v47.3.0 of @guardian/cdk](https://github.com/guardian/cdk/releases/tag/v47.3.0).